### ISSUE TITLE
What would be required to fix the plugin?

### ISSUE THREAD

**Issue by FDMatthias** [timdp/plugin.video.deredactie#2]
Hey,

first of all, thanks for making this. It was a sad day to find out it wouldn't work anymore as it was a plugin I used on a daily basis.

I briefly glanced through your code and thought I could help, but, it seems it all comes down to a certain type of url the plugin needs.. right? (It searches through some link elements in the atom files until it gets a url for the actual video, am I correct?)

Could you tell me what we're looking for exactly, or what would be needed to fix this? That way I know what to search and look for and maybe, just maybe, I could help..

Grtz,

Matthias

**Comment by timdp** [comment]
Indeed, so the issue is that VRT seem to have switched all their videos over to the VRT NU platform. They still produce the stream URLs that the plugin needs, but they're not providing those via the Atom feeds from deredactie.be anymore. Someone wrote a VRT NU plugin that's able to consume the feeds from there though, and Retrospect also supports them.

**Comment by FDMatthias** [comment]
Ah, well in that case, maybe it's better to continue with those plugins and
leave it at that.

For what it's worth, my gratitude for making this plugin. It had a good run :)
